nengo/spa/tests/test_bind.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+import nengo
+from nengo import spa
+
+
+def test_basic():
+    with spa.SPA() as model:
+        model.bind = spa.Bind(dimensions=16)
+
+    inputA = model.get_module_input('bind_A')
+    inputB = model.get_module_input('bind_B')
+    output = model.get_module_output('bind')
+    # all nodes should be acquired correctly
+    assert inputA[0] is model.bind.A
+    assert inputB[0] is model.bind.B
+    assert output[0] is model.bind.output
+    # all inputs and outputs should share the same vocab
+    assert inputA[1] is inputB[1]
+    assert inputA[1].dimensions == 16
+    assert output[1].dimensions == 16
+
+
+def test_run(Simulator, seed):
+    rng = np.random.RandomState(seed)
+    vocab = spa.Vocabulary(16, rng=rng)
+
+    with spa.SPA(seed=seed, vocabs=[vocab]) as model:
+        model.bind = spa.Bind(dimensions=16)
+
+        def inputA(t):
+            if 0 <= t < 0.1:
+                return 'A'
+            else:
+                return 'B'
+
+        model.input = spa.Input(bind_A=inputA, bind_B='A')
+
+    bind, vocab = model.get_module_output('bind')
+
+    with model:
+        p = nengo.Probe(bind, 'output', synapse=0.03)
+
+    sim = Simulator(model)
+    sim.run(0.2)
+
+    res = np.dot(vocab.parse("B*A").v, sim.data[p].T)
+
+    assert res[100] < 0.45
+    # significantly lower threshold than usual because of the noise
+    # involved in circular convolution
+    assert res[199] > 0.6
```

Comment:
I am wondering where the numbers 0.45 and 0.6 come from and whether these conditions are guaranteed?
If not, we could even use res[100] < res[199]